static/js/games/pong-game/pong.js

```diff
@@ -1,4 +1,345 @@
-import pongGameInit from './index.js'
+import * as THREE from 'three';
+import WebGL from 'three/addons/capabilities/WebGL.js'
+import { saveMatchResult } from "../save-match.js"
+
+function pongGameInit() {
+	GAME_RUNNING = true;
+	var RESOLUTIONS = [
+		{ w: 6400, h: 4800 },
+		{ w: 4096, h: 3072 },
+		{ w: 3200, h: 2400 },
+		{ w: 2048, h: 1536 },
+		{ w: 1152, h: 864 },
+		{ w: 1024, h: 768 },
+		{ w: 800, h: 600 },
+		{ w: 640, h: 480 },
+		{ w: 320, h: 240 },
+		{ w: 160, h: 120 },
+	]
```

Comment:
honestamente acho q isso aqui é inútil, eu tinha feito pq colocar pro canvas ser do tamanho do navegador deixava o jogo muito lento, mas agora q tirei a iluminação dinâmica acho q não precisa mais

Reply:
não tá usando isso?